CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# CoatRack Contributor Guidelines
+
+
+
+## Introduction
+
+#### Possibilities to Contribute
+
+Hello, dear prospective contributor. You can support CoatRack in many ways:
+
+- Report bugs using CoatRack's GitHub issue tracker.
+- Discuss suggested code changes in open Pull Requests.
+- Submit a bugfix.
```

Comment:
just to be consistent, all others are also plural:
```suggestion
- Submit bugfixes.
```

Reply:
Done.